### PR TITLE
Point from worker-manager to existing workertypes view

### PR DESCRIPTION
**Pull request by djmitche**
Per discussion with @owlishDeveloper and @helfi92

### CODE REVIEW

File: ui/src/components/Dashboard/SidebarList.jsx [hunk 1]
```diff
@@ -79,6 +79,10 @@ export default class SidebarList extends Component {
           Provisioners
         </SidebarListItem>
 
+        <SidebarListItem to="/worker-manager" icon={<MixcloudIcon />}>
+          Worker Manager
+        </SidebarListItem>
+
```
Comment (helfi92) [inline ui/src/components/Dashboard/SidebarList.jsx]: We would probably just need to remove `process.env.SHOW_WORKER_MANAGER`

https://github.com/taskcluster/taskcluster/blob/d6fddea345e02aa6075dc8cc80d89ddf47c949ce/ui/src/components/Dashboard/SidebarList.jsx#L72-L76

Comment (helfi92) [reply ui/src/components/Dashboard/SidebarList.jsx]: We will also need to remove `SHOW_WORKER_MANAGER` from `.neutrinorc.js`.

Comment (djmitche) [reply ui/src/components/Dashboard/SidebarList.jsx]: Oh, I didn't know that was a thing!

Comment (owlishDeveloper) [reply ui/src/components/Dashboard/SidebarList.jsx]: why to remove that variable? I don't want worker manager to be available in, well, not production, but the current public beta of the UI

File: ui/src/components/WMWorkerPoolsTable/index.jsx [hunk 2]
```diff
@@ -186,9 +190,9 @@ export default class WorkerManagerWorkerPoolsTable extends Component {
                 className={classes.viewWorkersButton}
                 variant="outlined"
                 component={Link}
-                to={`${this.props.match.path}/${encodeURIComponent(
```
Comment (owlishDeveloper) [inline ui/src/components/WMWorkerPoolsTable/index.jsx]: `${this.props.match.path}` was used to not hardcode paths

File: ui/src/components/Dashboard/SidebarList.jsx (hunk 1 above)
Comment (djmitche) [reply ui/src/components/Dashboard/SidebarList.jsx]: We're going to have people using it to set up workers in GCP, so it does need to be visible.

File: ui/src/components/WMWorkerPoolsTable/index.jsx (hunk 2 above)
Comment (djmitche) [reply ui/src/components/WMWorkerPoolsTable/index.jsx]: The new path goes all the way back to the root, so nothing of the matched route is preserved.  Is there a better way to generate the path for another view?

Comment (owlishDeveloper) [reply ui/src/components/WMWorkerPoolsTable/index.jsx]: So the `/provisioners/...` link leads to a different tool? The easiest thing to do is to hardcode then, in the absence of url constants

Comment (djmitche) [reply ui/src/components/WMWorkerPoolsTable/index.jsx]: It does.

**Pull request merged**